### ISSUE TITLE
dns-01: possible to check domains in parallel?

### ISSUE THREAD

**Issue by voltagex** [xenolf/lego#315]
I'm currently looking at 6 (sub)domains times ~30 minutes per validation. Would it be possible to create the TXT records for all requested domains and then check them in parallel?

**Comment by athei** [comment]
I support this. It would be awesome if lego would just create all the TXT records, wait for propagation of them all, then solve the challenge. That would be the way to go.

**Comment by boxofrox** [comment]
Duplicate of #173?  Looks like there's progress on this.

**Comment by ldez** [comment]
Fixed by #607
